### PR TITLE
[fix bug 1290313] update about page newsletter form

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Redesigned the newsletter subscription form on the about page in hopes it gets a bit more engagement. This also removes the photo mosaic in the header to speed up the page.

This update does include string changes but the form is only shown to English locales so I think it's safe.

Also made minor updates on a few other pages: removed the mosaic from /mission, fixed broken titles on forum pages.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1290313

### CODE REVIEW

**Comment by alexgibson** [comment]
Privacy link text colour needs updating:

privacy-link

File: bedrock/mozorg/templates/mozorg/about/manifesto.html [hunk 1]
```diff
@@ -310,14 +310,14 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
           {{ email_newsletter_form() }}
         </section>
       </div>
-      <div id="mosaic-group-1" class="ri-grid" data-loading="{{ static('img/mosaic/loading.gif') }}">
+      <div id="mosaic-group-1" class="ri-grid" data-loading="{{ static('img/about/manifesto/mosaic/loading.gif') }}">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/manifesto.html]: This img is 404 on the manifesto page. I think it should be `img/mozorg/about/manifesto/mosaic/loading.gif`

File: bedrock/mozorg/templates/mozorg/about/manifesto.html [hunk 2]
```diff
@@ -310,14 +310,14 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
...
         <ul>
         {% for i in range(1, 40)|list|shuffle %}
           <li><a><img data-src="{{ static('img/mozorg/about/manifesto/mosaic/' ~ '%03d'|format(i) ~ '.jpg') }}" alt=""></a></li>
         {% endfor %}
         </ul>
       </div>
-      <div id="mosaic-group-2" class="ri-grid" data-loading="{{ static('img/mosaic/loading.gif') }}">
+      <div id="mosaic-group-2" class="ri-grid" data-loading="{{ static('img/about/manifesto/mosaic/loading.gif') }}">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/manifesto.html]: Same as above

**Comment by alexgibson** [comment]
Now that we've removed the mosaic from the about & mission pages, we can delete [this test](mg/about/manifesto/mosaic/loading.gif) and [this test](https://github.com/mozilla/bedrock/blob/master/tests/functional/test_mission.py#L10-L14).

We can also delete the `is_mosaic_displayed` property and `_mosaic_locator` variables from both [about.py](https://github.com/mozilla/bedrock/blob/master/tests/pages/about.py) and [mission.py](https://github.com/mozilla/bedrock/blob/master/tests/pages/mission.py) as they will no longer be required.

**Comment by alexgibson** [comment]
A few small amends needed and some tests to tidy up, but otherwise this looks good. Yay for faster page loads! 🚀

**Comment by craigcook** [comment]
Updated!

**Comment by alexgibson** [comment]
r+ 🍍

File: bedrock/mozorg/templates/mozorg/about.html [hunk 3]
```diff
@@ -125,9 +119,9 @@ <h1 class="title-shadow-box">{{ _('Get to know Mozilla') }}</h1>
   <aside id="newsletter-subscribe">
     <div class="content container">
     {% if LANG.startswith('en-') %}
-      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='light') }}
+      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Sign up. Read up.<br> Make a difference.'), subtitle=_('Get the Mozilla newsletter and help us keep the Web free and open.'), button_class='red') }}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about.html]: cc @Peiying - a localizable string changed here, but it's only shown for `en-` locales.

Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/about.html]: Oops, wrong person. Should be @peiying2

**Pull request merged**